internal/server/server.go

```diff
@@ -100,7 +104,14 @@ func Serve(
 
 	headerMatcher := makeHeaderMatcher(config)
 
-	grpcConn, err := grpc.DialContext(grpcDialCtx, config.grpcAddr(), grpc.WithTransportCredentials(insecure.NewCredentials()))
+	grpcConn, err := grpc.DialContext(
+		grpcDialCtx,
+		config.grpcAddr(),
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+		grpc.WithDefaultCallOptions(
+			grpc.MaxCallRecvMsgSize(GRPCMaxClientSendSize),
```

Comment:
Do we need to increase RecvMsg as well? All of our insert APIs do not seem to need it.
wdyt?

Reply:
Not sure about this. 
Increased the `RecvMsg` as well, as asset payload can be huge - but haven't faced this scenario yet.